code/modules/mob/living/simple_animal/bot/hygienebot.dm

```diff
@@ -87,12 +87,6 @@
 	walk_to(src,0)
 	last_found = world.time
 
-/mob/living/simple_animal/bot/hygienebot/Moved()
```

Comment:
this was a specific request from maintainers to be added by the way

Reply:
(but i dont mind if its removed)

Reply:
Do you have a link? I don't think the requester understood the consequences. Try spawning one at a test server, bloodying yourself, and running around the central ring while it chases you, and you'll see.

Reply:
I'm not a maintainer, i just saw that it wet a tile after every few steps and thought that if it would be more consistent if it was based on Moved instead of a timer.

Reply:
I was talking about a comment on the discord

Reply:
which a maintainer made based on your comment I think